### PR TITLE
Homework5

### PR DESCRIPTION
**Pull request by parfenovaaa**

### CODE REVIEW

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Please review comments

File: src/test/java/com/epam/tc/hw5/cucumber/steps/UserTableSteps.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    @And("6 Description texts under images should be displayed on Users Table on User Table Page")
+    public void descriptionShouldBeDisplayed() {
+        Assert.assertTrue(userTablePage.sixElementsExist("descriptions"));
+    }
+
+    @And("6 checkboxes should be displayed on Users Table on User Table Page")
+    public void checkboxesShouldBeDisplayed() {
+        Assert.assertTrue(userTablePage.sixElementsExist("checkboxes"));
+    }*/
```
[the remark this answers is not in the document]
Comment (parfenovaaa) [reply src/test/java/com/epam/tc/hw5/cucumber/steps/UserTableSteps.java]: ok

File: src/test/resources/com/epam/tc/hw5/cucumber/Exercise1.feature [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    And I click on value 'Yellow' in the droplist on the Elements page
+    And I click on value 'Red' in the droplist on the Elements page
+    And I click on value 'Blue' in the droplist on the Elements page
+    Then Log screen should contain '4' lines:
+      | Log                             |
+      | Colors: value changed to Blue   |
+      | Colors: value changed to Red    |
+      | Colors: value changed to Yellow |
+      | Colors: value changed to Green  |
+
```
[the remark this answers is not in the document]
Comment (parfenovaaa) [reply src/test/resources/com/epam/tc/hw5/cucumber/Exercise1.feature]: Done. I just thought it`s similar to previous exercise.

**Review by akulebyakin — APPROVED** [review]
Approved

**Pull request merged**